profiler/kokoro/integration_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+retry() {
+  for i in {1..3}; do
+  [ $i == 1 ] || sleep 10  # Backing off after a failed attempt.
+    "${@}" && return 0
+  done
+  return 1
+}
+
+# Fail on any error.
+set -eo pipefail
+
+# Display commands being run.
+set -x
+
+cd $(dirname $0)/..
+
+export GOOGLE_APPLICATION_CREDENTIALS="${KOKORO_KEYSTORE_DIR}/72935_cloud-profiler-e2e-service-account-key"
+export GCLOUD_TESTS_GOLANG_PROJECT_ID="cloud-profiler-e2e"
+
+# Run test.
+retry go mod download
+retry go test -c
+./e2e.test -run_backoff_test=true
```

Comment:
This runs both the backoff test and the E2E test? What happens to the old E2E test configs? Do we still need them?

Reply:
Both the backoff test and E2E test will run.

The existing E2E test will still run as part of the E2E tests for this repo (and so haven't been changed). 

Does that seem reasonable?

Reply:
Could we run non-backoff E2E test only once, either removing it from here or from the E2E tests for this repo? Just don't want to maintain two configs for the same test and receive two alerts in case of error.

Reply:
Done. Only the backoff test will run now.